app/src/test/groovy/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettingsTest.groovy

```diff
@@ -0,0 +1,89 @@
+package gov.hhs.cdc.trustedintermediary.organizations
+
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.external.jackson.Jackson
+import gov.hhs.cdc.trustedintermediary.wrappers.formatter.Formatter
+import spock.lang.Specification
+
+import java.nio.file.Files
+import java.nio.file.Path
+
+class OrganizationsSettingsTest extends Specification {
+
+    Path tempFile
+    OrganizationsSettings settings
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+        TestApplicationContext.register(Organization, Organization)
```

Comment:
This doesn't need to be registered since it doesn't have anything to inject into it, nor is it injected into anything.